db.py

```diff
@@ -0,0 +1,180 @@
+import pathlib
+import sqlite3
+from abc import abstractmethod
+from os import getenv
+
+from enum import Enum
+from mysql.connector import connect, Error, ProgrammingError
+
+
+SCRIPT_DIR = pathlib.Path(__file__).parent.absolute()
+
+
+class DatabaseMixin:
+    @abstractmethod
+    def connect_to_database(self, database_name, database_exist):
+        raise NotImplementedError
+
+    @abstractmethod
+    def create_tables(self, queries):
+        raise NotImplementedError
+
+    @abstractmethod
```

Comment:
See comment below on how to make this method more generic: https://github.com/lorenanicole/eventbrite_scraper/pull/1/files#r658188614.